### ISSUE TITLE
Commends still not counted after profile update 

### ISSUE THREAD

**Issue by DrBroady** [ValveSoftware/Dota2-Gameplay#12257]
### Description

Multiple bug reports on this pre profile update. However, it's still not working when using the commend widget in the new profile system

**Comment by ligaapl** [comment]
Yes. I have 2172 a month ago and commends still not counted after profile update! 
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/146592265/fe1fe80e-1df1-4687-8b24-3bbd93b35f08)
